flyctl.hcl

```diff
@@ -1,7 +1,14 @@
 description = "Command line tools for fly.io services"
-binaries = ["flyctl"]
+binaries = ["flyctl", "fly"]
```

Comment:
Unfortunately this will break existing installations, because the "fly" binary won't exist in the bin directory.

You'll have to version this, so that historical versions have just flyctl, but future versions have the fly binary.